custom/icds_reports/reports/awc_reports.py

```diff
@@ -1221,7 +1223,7 @@ def base_data(row_data):
         return dict(
             case_id=row_data['case_id'],
             person_name=row_data['person_name'],
-            age=row_data['age_in_months'] // 12 if row_data['age_in_months'] else row_data['age_in_months'],
+            age=ceil((datetime.utcnow() - row_data['dob']).days / 365.25) if row_data['dob'] else row_data['dob'],
```

Comment:
why use `dob` instead of age in months?

Reply:
It was used, but reverted, due to another issue where changing age in the "edit member" form and running the aggregation script does not refresh age in ccs_record_cases.